src/mainwindow.cc

```diff
@@ -119,6 +122,30 @@ MainWindow::MainWindow():
     if (sourceTypeSelection.isValid())
         sourceTypeComboBox->setCurrentIndex(sourceTypeSelection.toInt());
 
+    const QVariant useSystray = config.value("systray/enabled");
```

Comment:
Did you consider using a default value like in other places (e.g. `config.value("systray/enabled", false).toBool()`)?

Reply:
Will fix that for clarity, to be honest I didn't notice default value in the line above that.